src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN =
+            "^(?=.{1,64}@)[A-Za-z0-9_-]+([.][A-Za-z0-9_-]+)*@[A-Z"
+            + "a-z0-9-]+([.][A-Za-z0-9-]+)*[.][A-Za-z]{2,}$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email == null) {
+            return false;
+        }
+        Pattern pattern = Pattern.compile(EMAIL_PATTERN);
```

Comment:
It is a good practice to compile patterns only once in such services. Let's create a constant of a type Pattern but not a String